modules/nf-core/tcoffee/alncompare/main.nf

```diff
@@ -0,0 +1,51 @@
+
+
+
+process TCOFFEE_ALNCOMPARE {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "bioconda::t-coffee=13.46.0.919e8c6b"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/t-coffee:13.45.0.4846264--hc57179f_5':
+        'biocontainers/t-coffee:13.45.0.4846264--hc57179f_5'}"
+
+    input:
+    tuple val(meta), path(msa), path(ref_msa)
+
+    output:
+    tuple val(meta), path("*.scores"), emit: scores
+    path "versions.yml"               , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = '-compare_mode tc' + task.ext.args ?: ''
```

Comment:
I think here we should first check if the field compare mode is present in args, if not add the -compare_mode_tc  otherwise it will always have the tc right?